roles/dtc/create/tasks/devices_discovery.yml

```diff
@@ -29,26 +29,31 @@
     save: true
     state: merged
   vars:
-      ansible_command_timeout: 3000
-      ansible_connect_timeout: 3000
+      ansible_command_timeout: 3600
+      ansible_connect_timeout: 3600
   when: MD_Extended.vxlan.topology.switches | length > 0
 
 - name: Create List of Switch Serial Numbers from Data Model
   ansible.builtin.set_fact:
     md_serial_numbers_list: "{{ MD_Extended.vxlan.topology.switches | map(attribute='serial_number') | list }}"
   delegate_to: localhost
 
-- name: Concatenate Switch Serial Numbers
-  ansible.builtin.set_fact:
-    md_serial_numbers: "{{ md_serial_numbers_list | join('%2C') }}"
-  delegate_to: localhost
-
-- name: Get NDFC Fabric Default Switch Policies
+- name: Get NDFC Fabric Default Switch Policies with temmplate name host_11_1
   cisco.dcnm.dcnm_rest:
     method: GET
-    path: "/appcenter/cisco/ndfc/api/v1/lan-fabric/rest/control/policies/switches?serialNumber={{ md_serial_numbers }}"
+    path: "/appcenter/cisco/ndfc/api/v1/lan-fabric/rest/control/policies/pagination?filter=templateName==host_11_1&fabricName={{ MD.vxlan.global.name }}"
```

Comment:
We cannot use this API as it's an internal-only API to NDFC and not publicly documented. It has potential to change without notice nor tracking thus not a good development practice to use. @dsx1123 @mikewiebe